docs-v2/spec.yaml

```diff
@@ -486,6 +486,96 @@ paths:
                                 properties:
                                     message:
                                         type: string
+    /connection/metadata:
```

Comment:
I like the simplicity of the endpoint, but a slight concern is that only the HTTP method (POST vs PATCH) differ when updating vs. overriding metadata. Users could make a mistake more easily and override metadata when they intended to update it, which can be disastrous.

But this might be over-defensive.

Reply:
This is exactly how it is now FWIW so this isn't a departure. I'd like to think we have enough warnings in the docs and a developer should take note of the HTTP method when performing operations.

Reply:
Ah got it, yes then there is not reason to change!